scripts/_generate/index.js

```diff
@@ -138,7 +138,7 @@ async function generateDownloads () {
 
     downloads.forEach(async function (dl) {
         console.log(`Generating ${dl.name}.tgz`);
-        let tmpDir = dl.name;
+        let tmpDir = `_tmp/${dl.name}`;
```

Comment:
prevents leftover `*.tgz` files in the root of the project (keeps temporary files contained within a subdirectory instead)